localnet/docker-compose.yml

```diff
@@ -248,7 +248,7 @@ services:
       start_period: 10s
 
   deposit-monitor:
-    image: ghcr.io/chainflip-io/chainflip-deposit-monitor/chainflip-deposit-monitor:${DEPOSIT_MONITOR_IMAGE_TAG:-main-4f97255318399222a92a9f03dfb7381f527b0184}
+    image: ghcr.io/chainflip-io/chainflip-deposit-monitor/chainflip-deposit-monitor:${DEPOSIT_MONITOR_IMAGE_TAG:-dev-b7cc56fed1ed1fecd8460b59f43ed038edfe649e}
```

Comment:
@MxmUrw is there a non-dev tag I can use for this now?

Reply:
We can use this tag for the time being, I'll put up a separate PR which updates the deposit-monitor and BLS tests.